### DISCUSSION TITLE
Kartenbild herunterladen // Größe

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#45] [Q&A]
Kann man die maximale Größe des herunterladbaren Kartenbildes einstellen, wenn ja wo? Momentan ist es ja mit 1920x1080 beschränkt.
![image](https://user-images.githubusercontent.com/1842500/222383117-774c4a86-4fbe-44f3-9039-31cf589401ac.png)

**Comment by gis-eni** [comment]
Die maximale Größe ist derzeit auf die Größe des Bildschrims des Anwenders beschränkt. Hier war die Idee, dass das Fenster nicht größer werden darf als der Bildschirm und vom Anwender keine beliebigen Werte eingeben werden können. 
Ein explizites Einstellen dieser Größe ist nirgends vorgesehen, weil sonst Anwender mit größerem Bildschirm immer das Problem haben, dass das Bild zu groß ist.

**Comment by gis-eni** [comment]
Möchte jemand höhere Auflösungen erreichen, ist Drucken das richtige Werkzeug.

Bild herunterladen ist eigentlich nur eine Art "Screenshot" Ersatz. Ändert man die Größe des Bildes manuell, ergibt sich ein komplett anderer Maßstab und das heruntergeladene Bild entspricht nicht mehr dem Bildschirminhalt.

**Comment by marko-jer** [comment]
Danke, sehe ich ein. Dann hätte ich ein Bitte, kann man das Feld "Bildgröße (Pixel)" bzw. auch die "BoundingBox" als readonly darstellen.

**Comment by gis-eni** [comment]
Mir grundsätzlich egal, die Frage ist, ob das irgendwer verwendet. Können wir beim nächsten WednesDay kurz diskutieren.
